README.md

```diff
@@ -43,7 +43,7 @@ Version Matrix
 
 Documentation
 --------------
-[Full documentation can be found here.](http://www.elasticsearch.org/guide/en/elasticsearch/client/php-api/5.0/index.html)  Docs are stored within the repo under /docs/, so if you see a typo or problem, please submit a PR to fix it!
+[Full documentation can be found here.](https://www.elastic.co/guide/en/elasticsearch/client/php-api/current/index.html)  Docs are stored within the repo under /docs/, so if you see a typo or problem, please submit a PR to fix it!
```

Comment:
I think is better to use `6.0` instead of `current` in the URL, since here the README is related to version 6.x of Elasticsearch. Current is related to the latest version, that will become 7.0 soon.

Reply:
tbh i think current is better because it will never be outdated, unless you have a checklist ready to change this link to 7.0 before releasing the next version.

Reply:
I'm actually planning to release a new 6.x soon, that's why I'm asking.